Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerImageAttachmentsView.swift

```diff
@@ -55,39 +47,39 @@ open class _ChatMessageComposerImageAttachmentsView<ExtraData: ExtraDataTypes>:
         collectionView.dataSource = self
         collectionView.delegate = self
         collectionView.register(
-            _ChatMessageComposerImageAttachmentCollectionViewCell<ExtraData>.self,
-            forCellWithReuseIdentifier: _ChatMessageComposerImageAttachmentCollectionViewCell<ExtraData>.reuseId
+            uiConfig.messageComposer.imageAttachmentCollectionViewCell.self,
+            forCellWithReuseIdentifier: uiConfig.messageComposer.imageAttachmentCollectionViewCell.reuseId
         )
     }
     
     override open func setUpLayout() {
-        collectionView.translatesAutoresizingMaskIntoConstraints = false
-        embed(collectionView)
-        
-        flowLayout.itemSize = imagePreviewItemSize
+        flowLayout.scrollDirection = .horizontal
+        flowLayout.itemSize = CGSize(width: 100, height: 100)
         flowLayout.sectionInset = layoutMargins
+
+        embed(collectionView)
     }
     
     override open func updateContent() {
         collectionView.reloadData()
     }
-
-    // MARK: - UICollectionView
     
     public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        images.count
+        content.count
     }
     
     public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        guard let cell = collectionView
-            .dequeueReusableCell(
-                withReuseIdentifier: _ChatMessageComposerImageAttachmentCollectionViewCell<ExtraData>.reuseId,
-                for: indexPath
-            ) as? _ChatMessageComposerImageAttachmentCollectionViewCell<ExtraData>
-        else { return UICollectionViewCell() }
-        
-        cell.imageView.image = images[indexPath.row]
-        cell.discardButtonHandler = { [weak self] in self?.didTapRemoveItemButton?(indexPath.row) }
+        let reuseId = uiConfig.messageComposer.imageAttachmentCollectionViewCell.reuseId
+        let cell = collectionView.dequeueReusableCell(
+            withReuseIdentifier: reuseId,
+            for: indexPath
+        ) as! _ChatMessageComposerImageAttachmentCollectionViewCell<ExtraData>
```

Comment:
ok I guess this solves the logging issue :trollface:

Reply:
Ahah, yes I did this because we are doing this in all other places. So it is to keep consistency. And ofc this will only happen if the dev made a mistake, so no big deal